### ISSUE TITLE
Include color modifies code example in modifier docs

### ISSUE THREAD

**Issue by six7** [tokens-studio/plugin-docs#73]
We should show how color modifiers look/work in JSON by showing it in the modifier docs.

**Comment by isaquepereira** [comment]
Yes please. I'm currently RN search about LCH to understand how it works and document my project.
If anyone has a hint, please share with me!
